submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/cluster/web/ClusterOperationsResource.kt

```diff
@@ -1,22 +1,45 @@
 package ac.uk.ebi.biostd.cluster.web
 
-import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
+import arrow.core.Try
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
 import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
 import uk.ac.ebi.biostd.client.cluster.api.ClusterOperations
 import uk.ac.ebi.biostd.client.cluster.model.Job
 import uk.ac.ebi.biostd.client.cluster.model.JobSpec
+import uk.ac.ebi.biostd.client.cluster.model.MemorySpec
+import uk.ac.ebi.biostd.client.cluster.model.QueueSpec
 
 @RestController
-@ConditionalOnProperty(prefix = "app.task", name = ["enableTaskMode"], havingValue = "true")
 @RequestMapping("/cluster")
 class ClusterOperationsResource(
     private val clusterOperations: ClusterOperations,
 ) {
     @PostMapping("/health")
     suspend fun clusterHealthCheck(): Job {
         val spec = JobSpec(command = "echo 'hello world'")
-        return clusterOperations.awaitJob(spec)
+        return clusterOperations.runJob(spec)
+    }
+
+    @PostMapping("/submit")
+    suspend fun submitJob(@RequestBody job: JobSpecDto): Job {
+        return when (val response = clusterOperations.triggerJob(job.asJobSpec())) {
+            is Try.Failure -> throw IllegalStateException(response.exception)
+            is Try.Success -> response.value
+        }
+    }
+
+    @GetMapping("/jobs/{jobId}/status")
+    suspend fun jobStatus(@PathVariable jobId: String): String {
+        return clusterOperations.jobStatus(jobId)
+    }
+
+    data class JobSpecDto(val command: String, val queue: String, val megaBytes: Int) {
```

Comment:
I think the last parameter should be called `memory`, `ramSize` or even just `ram` like in the Job class because `megaBytes` is not very clear at communicating what the value is actually used for